mobile_index/Cargo.toml

```diff
@@ -0,0 +1,52 @@
+[package]
+name = "mobile-index"
+version = "0.1.0"
+description = "Reward Indexer for the Helium Mobile Network"
+edition.workspace = true
+authors.workspace = true
+license.workspace = true
+
+
+[dependencies]
+config = {workspace = true}
+dotenv = {workspace = true}
```

Comment:
How did `dotenv` sneak back in there? I thought you removed its usage.

Reply:
yeah, odd.. removed and trimmed the mobile_index member source since it's not doing anything quite yet as we migrate config approach